src/aeonlib/ocs/blanco/instruments.py

```diff
@@ -1,17 +1,32 @@
 # pyright:  reportUnannotatedClassAttribute=false
 # This file is generated automatically and should not be edited by hand.
 
-from typing import Any, Annotated, Literal
+from typing import Any, Annotated, Literal, Optional
 
-from annotated_types import Le
-from pydantic import BaseModel, ConfigDict
+from annotated_types import Le, Ge
+from pydantic import BaseModel, ConfigDict, Field
 from pydantic.types import NonNegativeInt, PositiveInt
 
 from aeonlib.models import TARGET_TYPES
 from aeonlib.ocs.target_models import Constraints
 from aeonlib.ocs.config_models import Roi
 
 
+
+class BlancoNewfirmConfigExtraParams(BaseModel):
+    model_config = ConfigDict(validate_assignment=True, extra='allow')
+    dither_value: Annotated[int, Ge(0), Le(1600)] = 80
+    dither_sequence: Literal[['2x2', '3x3', '4x4', '5-point']] = "2x2"
```

Comment:
Invalid syntax, There's an extra set of [] in there.

Reply:
oops, ill remove that